packages/lib-classifier/src/store/utils/ClassificationQueue.spec.js

```diff
@@ -109,14 +109,14 @@ describe('ClassificationQueue', function () {
   })
 
   describe('keeps classifications in localStorage if backend fails', function () {
-    let clock
-
     before(function () {
-      clock = sinon.useFakeTimers({ global })
+      sinon.stub(global, 'setTimeout').callsFake(() => 100)
+      sinon.stub(global, 'clearTimeout')
```

Comment:
@eatyourgreens the use of `sinon.useFakeTimers()` was added in https://github.com/zooniverse/front-end-monorepo/pull/3019. I'm not totally sure of the difference between `.setFakeTimers()` vs `.stub(global` here, but reverted to `.stub()` allows the tests to pass. Any thoughts on this sinon method?

Reply:
The docs for fake timers are here but, as noted, it’s just a thin wrapper for the Fake Timers package. These docs do explain `config.global`. 

https://sinonjs.org/releases/latest/fake-timers/

Reply:
With your change, the tests run using the local clock ie. the clock continues to tick during the tests. With fake timers, I think the test clock doesn’t tick unless you tell it to tick.

Reply:
This test now looks like it’s evergreen to me, because timeouts can’t be set by the mocks. If there were bugs in the timeout code, I think the mocks might hide them. 

https://github.com/zooniverse/front-end-monorepo/blob/2de1f3e91a26d75d50a54b7291cf562020fd3842/packages/lib-classifier/src/store/utils/ClassificationQueue.spec.js#L206-L210